susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- Add 'python-singledispatch' to SLE12+ and RES7 bootstrap repos.
```

Comment:
Why SLE12+? We don't add it for SLE15, right?

Reply:
fixed